website/docs/reference/artifacts/run-results-json.md

```diff
@@ -30,7 +30,7 @@ Note: `dbt source freshness` produces a different artifact, [`sources.json`](/re
 
 Each entry in `results` is a [`Result` object](/reference/dbt-classes#result-objects), with one difference: Instead of including the entire `node` object, only the `unique_id` is included. (The full `node` object is recorded in [`manifest.json`](/reference/artifacts/manifest-json).)
 
-- `unique_id`: Unique node identifier, which map results to `nodes` in the [manifest](/reference/artifacts/manifest-json)
+- `unique_id`: Unique node identifier, which maps results to `nodes` in the [manifest](/reference/artifacts/manifest-json)
```

Comment:
Looks good! @ialdg  You can add your comment by clicking the + sign next to your fix!
> [comment]: To correct a typo at the "unique_id" bulletpoint.